<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>671</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>로그 추가</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>290</y>
       <width>181</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>상세보기</string>
     </property>
     <widget class="QLabel" name="labelDetail">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>161</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>481</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>삽입</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>460</width>
        <height>195</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="3">
        <widget class="QCheckBox" name="checkDefault">
         <property name="text">
          <string>현재시간</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTimeEdit" name="editTime">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumDateTime">
          <datetime>
           <hour>15</hour>
           <minute>59</minute>
           <second>57</second>
           <year>2000</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="minimumDateTime">
          <datetime>
           <hour>9</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2000</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>매수시간</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboKind">
         <item>
          <property name="text">
           <string>레버리지</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>인버스</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>종목명</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>가격</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>비고</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>수량</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QPushButton" name="btnPlusTime">
           <property name="text">
            <string>+5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMinusTime">
           <property name="text">
            <string>-5</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLineEdit" name="lineCost"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>원</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QPushButton" name="btnPlusCost">
           <property name="text">
            <string>+5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMinusCost">
           <property name="text">
            <string>-5</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLineEdit" name="lineAmount"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>개</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QPushButton" name="btnPlusAmount">
           <property name="text">
            <string>+5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMinusAmount">
           <property name="text">
            <string>-5</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QTextEdit" name="textDesc"/>
       </item>
       <item row="1" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>구분</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBuySell">
           <item>
            <property name="text">
             <string>매수</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>매도</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btnAdd">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>5</horstretch>
           <verstretch>5</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>추가</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>291</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>매수/매도목록</string>
     </property>
     <widget class="QListWidget" name="listProduct">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>211</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDelete">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>삭제</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMakeFile">
      <property name="geometry">
       <rect>
        <x>229</x>
        <y>45</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>만들기</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>프로그램 종료</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>493</x>
       <y>30</y>
       <width>20</width>
       <height>491</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <zorder>groupBox</zorder>
    <zorder>btnAdd</zorder>
    <zorder>groupBox_3</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>gridLayoutWidget_2</zorder>
    <zorder>line</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>로그 확인</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>파일목록</string>
     </property>
     <widget class="QListWidget" name="listFile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>151</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDfile">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>파일삭제</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOutput">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>챠트보기</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>textDesc</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
